fedmsg_meta_fedora_infrastructure/autocloud.py

```diff
@@ -85,7 +86,10 @@ def handle_v2_autocloud_subtitle(self, msg, **config):
                                 compose_id=compose_id)
 
         if "autocloud.compose" in msg['topic']:
-            compose_id = msg['msg']['id']
+            compose_id = msg['msg'].get('id', '')
+
+            if compose_id is '' and "complete" in msg['topic']:
+                compose_id = msg['msg'].get('compose_id', '')
 
```

Comment:
And if none of them are present?

With this code we will not have an error but the message returned will also not be understandable. So should we crash so that we are at least notified of an odd message?

The part that worries me a little is:
* Retrieve ``id``
* If there is no ``id`` *and the topic contains ``complete``*: retrieve compose_id

So if there is no ``id`` but the topic does not contain ``complete`` then ``compose_id`` remains ``''``